index.html

```diff
@@ -722,26 +702,49 @@ <h3>Human Readability</h3>
     </section>
 
     <section>
-      <h2>Cryptography</h2>
-
+      <h2>Proof Formats</h2>
       <em>This section is non-normative.</em>
 
       <p>
+The <a>verifiable credentials</a>
+<a href="https://w3c.github.io/vc-data-model/">data model </a> is designed to be
+proof format agnostic. <a href="https://w3c.github.io/vc-data-model/">The
+specification</a> does not normatively require any particular digital proof or
+signature format. While the data model is the canonical representation of a
+<a>verifiable credential</a> or <a>verifiable presentation</a>, the proving
+mechanisms for these are often tied to the syntax used in the transmission of
+the document between parties. As such, each proofing mechanism has to specify
+whether the validation of the proof is calculated against the state of the
+document as transmitted, against the transformed data model, or against another
+form. At the time of publication, at least two proof formats are being actively
+utilized by implementers and the Working Group felt that documenting what these
```

Comment:
`implementers and`
-> `implementers, and`

Reply:
fixed in [9a0eaac](https://github.com/w3c/vc-imp-guide/pull/28/commits/9a0eaac6b21a3c4c4a8281809a1e809f781aadfc)